src/librustc_trans/back/msvc/mod.rs

```diff
@@ -251,7 +252,16 @@ pub fn link_exe_cmd(sess: &Session) -> Command {
             }).map(|path| {
                 path.join("um").join(extra)
             })
-        }
+        } else if let Some(mut ver) = ver {
+            // See comment above
```

Comment:
This may want to be expanded because the logic here is fairly different from 8.x